test/lib/square_checkout_test.rb

```diff
@@ -72,29 +81,13 @@ class SquareCheckoutTest < ActiveSupport::TestCase
     mock_response.expect :success?, false
     mock_response.expect :errors, "ERRORS"
 
-    mock_checkout = Minitest::Mock.new
-    mock_checkout.expect(:create_checkout, mock_response,
-      location_id: "SQ_LOCATION_ID",
-      body: {
-        idempotency_key: "test",
-        redirect_url: "http://example.com/callback",
-        pre_populate_buyer_email: member.email,
-        order: {
-          order: {
-            location_id: "SQ_LOCATION_ID",
-            reference_id: "20210101-#{member.id}",
-            line_items: [{
-              name: "Annual Membership",
-              quantity: "1",
-              base_price_money: {
-                amount: form.amount.cents,
-                currency: "USD"
-              }
-            }]
-          }
-        },
-        note: "Circulate signup payment"
-      })
+    # No need to assert anything about what params are passed to
+    # this method since we do that in the successful case. There doesn't seem to
+    # be a way to assert that a mocked method is called with _any_ kwargs.
```

Comment:
I just ran into this too w/ the new minitest kwargs support: you can't do _any_ kwargs but you can get almost there. The matchers run per key but they use `===` so if you know the keys you can specify keys w/ class names as values  to allow any instance of that class as a value. [Here's an example](https://github.com/chicago-tool-library/circulate/pull/1291/files#diff-c87f8805a0f4917ed3ab9cfc15eb237da4c2e3d2dcb8f658701687c8cbe64fe1R103) from a test I had to fix with the upgrade.

Reply:
Oh, nice. I'll give that a shot- I'd like to assert something but really want to avoid it being too specific.